tests/kvmtest.sh

```diff
@@ -92,6 +92,8 @@ RUNTEST_CLI_COMMON_OPTS="\
 -e --allow_recover \
 -e --completeness_level=confident"
 
+MULTI_ASIC_CLI_OPTIONS=`echo $RUNTEST_CLI_COMMON_OPTS | sed 's/-q 1//g'`
```

Comment:
Can you add a comment here explaining that this would continue upon a single test failure.

Reply:
done